### PR TITLE
ONI-83: Prevent increase of item/service quantity during review.

### PR DESCRIPTION
**Pull request by wzglinieckisoldevelo**
Ticket: https://openimis.atlassian.net/browse/ONI-83

Changes:
- Saving a review is blocked if user tries to increase item/service quantity above original value
- Warning is shown under this field

claim_review_max_quantity

### CODE REVIEW

File: src/components/ClaimForm.js [hunk 1]
```diff
@@ -179,7 +184,7 @@ class ClaimForm extends Component {
       if (!!this.state.claim.items) {
         items = [...this.state.claim.items];
         if (!this.props.forReview) items.pop();
-        if (items.length && items.filter((i) => !this.canSaveDetail(i, "item")).length) {
+        if (items.length && items.filter((i) => !this.canSaveDetail(i, "item", forReview=forReview)).length) {
```
Comment (olewandowski1) [inline src/components/ClaimForm.js:187]: It should be enough to pass forReview here, no need to pass forReview=forReview.

File: src/components/ClaimForm.js [hunk 2]
```diff
@@ -193,7 +198,7 @@ class ClaimForm extends Component {
           }
         }
         if (!this.props.forReview) services.pop();
-        if (services.length && services.filter((s) => !this.canSaveDetail(s, "service")).length) {
+        if (services.length && services.filter((s) => !this.canSaveDetail(s, "service", forReview=forReview)).length) {
```
Comment (olewandowski1) [inline src/components/ClaimForm.js:201]: It should be enough to pass forReview here, no need to pass forReview=forReview.

**Review by olewandowski1 — CHANGES_REQUESTED** [review]
I left some comments. @wzglinieckisoldevelo

**Review by olewandowski1 — APPROVED** [review]
LGTM @wzglinieckisoldevelo

**Review by sniedzielski — APPROVED** [review]

**Review by dborowiecki — APPROVED** [review]

**Pull request merged**